docs/developer/centrifuge-sdk/buying-or-selling-defi-assets/index.md

```diff
@@ -0,0 +1,139 @@
+# Buy and sell DeFi assets
+
+This guide shows how to use the Centrifuge SDK to buy and sell tokenized DeFi assets within a pool.
+
+## Prerequisites
+
+Before you begin, make sure you have:
+
+- [Node.js](https://nodejs.org/) (v18 or later recommended)
+- A package manager: [npm](https://www.npmjs.com/) or [yarn](https://yarnpkg.com/)
+- A wallet or signer that can connect to Ethereum-compatible chains (e.g. MetaMask, WalletConnect, or a private key account via [Viem](https://viem.sh/))
+
+## Installation
+
+Install the Centrifuge SDK in your project:
+
+```bash
+pnpm add @centrifuge/sdk
+```
+
+## 1. Initialize the SDK
+
+Create a Centrifuge instance and connect it to mainnet:
+
+```typescript
+import Centrifuge from "@centrifuge/sdk";
+
+const centrifuge = new Centrifuge({
+  environment: "mainnet",
+});
+```
+
+:::info
+For testing purposes, you can connect to testnet instead by setting environment: `testnet`.
+:::
+
+## 2. Deploy MerkleProofManager
+
+```typescript
+const poolId = new PoolId(1);
+const pool = await centrifuge.pool(poolId);
+const scId = ShareClassId.from(poolId, 1);
+
+const poolNetworks = await pool.activeNetworks();
+
+await poolNetwork.deployMerkleProofManager();
+```
+
+## 3. Retrieve MerkleProofManager and add as balance sheet manager
+
+Retrieve the deployed MerkleProofManager and set it as a BalanceSheet manager:
+
+```typescript
+const merkleProofManager = await poolNetwork.merkleProofManager();
```

Comment:
It's not clear above how to retrieve a specific `poolNetwork`, only the list of all pool networks is loaded